src/main/java/com/dubture/jenkins/digitalocean/Utils.java

```diff
@@ -0,0 +1,202 @@
+package com.dubture.jenkins.digitalocean;
+
+import com.myjeeva.digitalocean.exception.DigitalOceanException;
+import com.myjeeva.digitalocean.exception.RequestUnsuccessfulException;
+import com.myjeeva.digitalocean.impl.DigitalOceanClient;
+import com.myjeeva.digitalocean.pojo.Droplet;
+import com.myjeeva.digitalocean.pojo.Droplets;
+import com.myjeeva.digitalocean.pojo.Image;
+import com.myjeeva.digitalocean.pojo.Images;
+import com.myjeeva.digitalocean.pojo.Key;
+import com.myjeeva.digitalocean.pojo.Keys;
+import com.myjeeva.digitalocean.pojo.Region;
+import com.myjeeva.digitalocean.pojo.Regions;
+import com.myjeeva.digitalocean.pojo.Size;
+import com.myjeeva.digitalocean.pojo.Sizes;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.SortedMap;
+import java.util.TreeMap;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static com.google.common.collect.Lists.newArrayList;
+
+/**
+ * Various utility methods that make it easier to obtain full lists of properties from Digital Ocean. Some API
+ * calls require page number, since the results are paginated, so these utilities will exhaust all the pages and
+ * return a single result set.
+ *
+ * @author Rory Hunter (rory.hunter@blackpepper.co.uk)
+ */
+public class Utils {
```

Comment:
Utils is kind of the wrong name for this class. It's rather DigitalOcean as static class